Automattic/jetpack
# Vdimitrakis/selfserve 705/white list dsp api experiments call

Fixes #
please check SELFSERVE-705

## Proposed changes:
Added a rest route endpoint to handle the dsp/api/experiments API call 

### Other information:


## Jetpack product discussion



## Does this pull request change what data or activity we track or use?



## Testing instructions:






Fire up your local jetpack installation (and tunnel it - so that advertising page is working properly)
navigate to tools->advertising and click to promote a post (or page) 
open up your network inspector and verify that the call to /experiments is going through without an error 

the full path should be something like the one below (with different ids)

wp-json/jetpack/v4/blaze-app/sites/221890190/wordads/dsp/api/v1/experiments/assignments/29032023



(in trunk this call should be failing)
